extension.js

```diff
@@ -111,8 +98,16 @@ function enable() {
         return;
     }
 
-    if (run('enable'))
+    SESSIONID += "#";
+    Extension = imports.misc.extensionUtils.getCurrentExtension();
+    warnAboutGnomeShellVersionCompatibility();
```

Comment:
Oh good point - yeah, we should actually use in functions!

Reply:
Yes, I think so.  The other option is remove this check / warning entirely.  `metadata.json` now is stricter in versions supported, e.g. if it's not running `42`, `43`, or `44` you won't be able enable this extension.

The only advantage I can see with keeping this is if we enable early/prelim support for `45` - although, lessons from `44` issues suggest we should just do that in a `45` branch for users that are on `45` to test etc.

Reply:
So, I pushed some changes to: only run `warnAboutGnomeShellVersionCompatibility` on first enable (and not subsequent enable/disable calls).

Did this as an interim until we decide if we want to keep this check (or if you can think of other use/corner cases where this check woudl be needed).

Reply:
Yes, I think now with the new `release` branch we can also remove this warning. Because now people will not as easily accidentally update paperwm to an experimental state.